data-access-analysis-tools/index-rsp-warnings.rst

```diff
@@ -12,98 +12,60 @@
-
-Users installing code: DP0 delegates will be able to add software that can be installed without privileges into their own homespace (eg. via pip install --user) for their personal use to the extent that their quota allows.
-
-
-Bulk Downloads are not Supported
---------------------------------
-
-Bulk downloads: No, bulk data downloads from the RSP specifically, or the IDF in general, are not supported during DP0. This is because bulk download services are not yet in production, and because the major goals of DP0 are to allow Rubin Observatory to evaluate how the community uses the services, and for the community to become familiar with the RSP environment. Note that DESC has made the DC2 DR6 catalogs publicly available via: https://lsstdesc-portal.nersc.gov/.
-
-
-
-Stuff That We Know Does Not Work
---------------------------------
-
-*(Here is space for a list of known issues and stuff that doesn’t work in , e.g., the portal. Especially the portal. To avoid people thinking “This is too hard to figure out” when really it’s just a feature that’s not done.)*
-
-ADQL functionality that does not yet work: 
-AREA
-BOX
-COORDSYS
-COORD1
-COORD2
-INTERSECTS
-
-The only coordinate system currently supported is 'ICRS'.
+- Notebooks: We are planning on providing a filespace that you can access from your personal device (eg laptop) so you can use your favourite editor/IDE to write code in your local environment and save it in a way that makes it visible in your RSP notebook environment
+- Batch: Opportunities for parallelized/batch/non-interactive computation are not currently available
+- Qserv: User tables are not yet available
+- Qserv: TAP/ADQL Queries using some keywords (AREA, BOX, COORDSYS, COORD1, COORD2 and INTERSECTS) are not yet supported
+- Portal: Many improvements are planned, including the ability to start a query in the portal and access it from your notebook
+- Authentication and Authorisation: You will be able to create and manage groups to allow you to share data with specific RSP users
+- API: Virtual Observatory image services are not yet available
+- API: VOSpace service is not yet available
+- ConsolidatedDB: Access to observatory metadata is not yet available
+- Bulk Download Service: Not offered
 
+Warnings and admonishments
+---------------------------
 
-MLG Did Not Know The Category
------------------------------
+Our `Acceptable Use Policy <https://data-dev.lsst.cloud/terms>`_ applies: We’re giving you access to Rubin Observatory systems so you can do science with the data we provide or otherwise further the mission of the observatory. You can lose your access if you misuse our resources, interfere with other users, or otherwise do anything that would bring the observatory into disrepute.
```

Comment:
I would capitalize "Observatory" throughout.

Reply:
Ok, also fixed in https://github.com/lsst-sqre/squareone/pull/23